lib/awful/hotkeys_popup/keys/tmux.lua

```diff
@@ -0,0 +1,73 @@
+---------------------------------------------------------------------------
+--- tmux hotkeys for awful.hotkeys_widget
+--
+-- @autor nahsi nashi@airmail.cc
+-- @copyright 2017 nahsi
+-- @module awful.hotkeys_popup.keys.tmux
+---------------------------------------------------------------------------
+
+local hotkeys_popup = require("awful.hotkeys_popup.widget")
+
+local tmux_rule = { name = "tmux" }
```

Comment:
I think it should pass in the whole of `group_data` then, so that you can use `rule_any` etc.

I am using the following to match the main tmux session for the current desktop / X session with my run-or-raise:
```lua
        { rule_any = {
          name = {string.format('^%%[%s:.*:tmux%%].*', tmux_session),
                  string.format('^%%[%s:.*:tmux%%].*', 'some-project')}
          },
          except_any = { name = { "^%[weechat:",
                                  "^%[IGNORE:" } },
        },
        matcher=awful.rules.matches
```

That might not be really relevant, but would allow for more flexibility in general.

For my use case `{ rule = { name = "^%[tmux: .*" }` would work in general, but would not match the "weechat" tmux session then.